src/virtual-machine.js

```diff
@@ -345,11 +351,21 @@ class VirtualMachine extends EventEmitter {
         });
         
         this.runtime.on('MQTT_ALIAS_VAR_INBOUND', data => {
-            this.setAliasVariables(data);
+            if (this.standaloneMode === true) {
+                this.setStandaloneAliasVariables(data);
+            }
+            if (this.standaloneMode === false) {
+                this.setAliasVariables(data);
+            }
```

Comment:
if (this.standaloneMode) { } else { }

Reply:
if {} else {} syntax actually goes against the Scratch linter controls